apps/testingaccessibility/src/components/pricing.tsx

```diff
@@ -188,7 +188,7 @@ export const Pricing: React.FC<React.PropsWithChildren<PricingProps>> = ({
           </div>
         ) : (
           <form
-            action={`/api/stripe/checkout?productId=${
+            action={`/api/skill/checkout/stripe?productId=${
```

Comment:
the `checkout` is the "action" and `stripe` is the provider. Technically we'd be able to utilize different providers in the future but that's not in yet.